db/migrate/20200727124429_create_comments.rb

```diff
@@ -0,0 +1,10 @@
+class CreateComments < ActiveRecord::Migration[5.2]
+  def change
+    create_table :comments do |t|
+      t.string :commenter
+      t.string :text
```

Comment:
Why all these fields?? Please, re-read the requirements to know the required fields

Reply:
they are generated migration, validations are only done on the models

Reply:
fixed